.storybook/stories/useFBXLoader.stories.js

```diff
@@ -0,0 +1,41 @@
+import React, { Suspense } from 'react'
+
+import { Setup } from '../Setup'
+
+import {useFBXLoader} from "../../src/loaders/useFBXLoader";
```

Comment:
Looks like there are some style issues here, can you make sure the code is ran through prettier?